src/routes/securityQuestions.ts

```diff
@@ -0,0 +1,14 @@
+import { Router } from 'express';
+import * as securityQuestionController from '../controllers/securityQuestionController';
+import { adminAuth } from '../middleware/auth';
+
+const router = Router();
+
+router.get('/', securityQuestionController.indexSecurityQuestionFuncs);
```

Comment:
Unused routes not removed here

Reply:
This route is to get all the security questions though (active and non-active)